data/src/main/java/io/homeassistant/companion/android/data/url/UrlRepositoryImpl.kt

```diff
@@ -21,7 +21,8 @@ class UrlRepositoryImpl @Inject constructor(
         private const val PREF_REMOTE_URL = "remote_url"
         private const val PREF_WEBHOOK_ID = "webhook_id"
         private const val PREF_LOCAL_URL = "local_url"
-        private const val PREF_WIFI_SSID = "wifi_ssid"
+        private const val PREF_WIFI_SSID = "wifi_ssid" // Keep for backwards compatibility
```

Comment:
Rather than keep it around in live code we should try to use the https://github.com/home-assistant/home-assistant-android/blob/master/common/src/main/java/io/homeassistant/companion/android/common/migrations/Migrations.kt.